torch/optim/adam.py

```diff
@@ -83,8 +83,10 @@ def _init_group(
                 state = self.state[p]
                 # Lazy state initialization
                 if len(state) == 0:
+                    # note(crcrpar): Deliberately host `step` on CPU if both capturable and fused are off.
+                    # This is because kernel launches are costy on CUDA and XLA.
                     state['step'] = (
-                        torch.zeros((1,), dtype=torch.float, device=p.device)
+                        torch.tensor(0.0, dtype=torch.float, device=p.device)
```

Comment:
We can keep this as torch.zeros right? Or is there actual significance in using tensor
```suggestion
                        torch.zeros((), dtype=torch.float, device=p.device)
```

Reply:
Is this mainly for parity with the else case?